### DISCUSSION TITLE
Is there a way to disable matching brackets and quotes?

### DISCUSSION THREAD

**Discussion by madeinouweland** [zed-industries/zed#6611] [Feature Requests]
Zed matches opening brackets and quotes by adding or removing their closing counterpart automatically. But this often does not do what I intended and I usually end up with too many or not enough brackets/quotes.
Is there a way to disable 'matching brackets and quotes' in Zed or are there plans to do it?

**Comment by hez** [comment]
This!

**Comment by JosephTLyons** [comment]
We don't have a setting for this at the moment, but we should. There's an issue that sort of is asking for the same thing.

- https://github.com/zed-industries/community/issues/1903

It doesn't help that we have a bit of buggy behavior when closing quotes, where multiple quotes are inserted when only one is expected. We should fix that too.

**Comment by notpeter** [comment]
Apologies for the necromancy.
This is now supported (either globally or per-language):
 
```json
  "use_autoclose": false,
```

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
